etc/config/cuda.amazon.properties

```diff
@@ -228,6 +228,12 @@ compiler.nvcc123u1.options=--compiler-bindir /opt/compiler-explorer/gcc-10.2.0/b
 compiler.nvcc123u1.nvdisasm=/opt/compiler-explorer/cuda/12.3.1/bin/nvdisasm
 compiler.nvcc123u1.demangler=/opt/compiler-explorer/gcc-10.2.0/bin/c++filt
 compiler.nvcc123u1.objdumper=/opt/compiler-explorer/gcc-10.2.0/bin/objdump
+compiler.nvcc124u1.semver=12.4.1
```

Comment:
`nvcc124u1` needs to be added to line 12 of this file

Reply:
So what's the best way to resolve this?  Delete the existing PR and submit a new one?

Reply:
no need, you can just edit the file in github and push it that way